arch-design/multicloud/BucketManagement.md

```diff
@@ -105,6 +116,86 @@ Below diagram shows two things:
 
 
 ### SODA-API for Bucket management:
+******
+#### Tier policy API
+******
+
+#### Create Tier:
+
+##### [POST /HTTP/v1/tier-name]()
+
+###### Request Body:
+`tier_name`: string, Name of tier
+
+`backend`: string, Name to backend
+
+###### Response:
+
+ReturnCode: integer, Ex:200
+
+Message: string, Ex: Successfully Created!
+
+
+#### List all Tiers:
+
+##### [GET /HTTP/v1/tiers]()
+
+###### Request Body:
+N/A
+
+###### Response:
+
+ReturnCode: integer, Ex:200
+
+Message: [], Ex: [Gold]
+
+
+#### Get Tier:
+
+##### [GET /HTTP/v1/tiers/{tier}]()
+
+###### Request Body:
+N/A
+
+###### Response:
+
+ReturnCode: integer, Ex:200
+
+Message: json
+
+
+#### Update Tier:
+
+##### [PUT /HTTP/v1/{tier_name}]()
+
+###### Request Body:
+`tier_name`: string, Name of tier
+
+`backend`: string, Name to backend
+
+###### Response:
+
+ReturnCode: integer, Ex:200
+
+Message: string, Ex: Successfully Updated!
+
+
+#### Delete Tier:
+
+##### [DELETE /HTTP/v1/{tier_name}]()
```

Comment:
there can be multiple backends with same tier name, right? will this delete for all backends?